ts/Core/Tooltip.ts

```diff
@@ -1723,8 +1734,18 @@ class Tooltip {
             return boxes;
         }, []);
 
+        const isInsideRightBounds = (box: Record<string, any>): boolean => (
+            (tooltip.outside ?
+                box.anchorX + distance + box.boxWidth :
+                box.point.plotX + box.boxWidth) < bounds.right
```

Comment:
The left-overflow test simply tests for `box.x < scrollLeft`. Why is the right-overflow test more complicated? I would expect `box.x + box.boxWidth` to reveal the right edge of the tooltip regardless of `tooltip.outside`, since `box.x` is used regardless of `tooltip.outside` in the left-overflow check. If so, it would be easier to read and maintain this code if the left and right checks looked the same.